ciao-controller/internal/datastore/datastore.go

```diff
@@ -643,7 +644,6 @@ func (ds *Datastore) GetAllTenants() ([]*types.Tenant, error) {
 	// numbers of tenants. I'd rather reuse the code
 	// than make this more efficient at this point.
 	ts, err := ds.getTenants()
-
 	if err != nil {
 		return nil, err
```

Comment:
Ok makes sense.  ds.getTenants() either does something that can't fail or calls ds.db.getTenants(), where the latter would do an errors.Wrap().